### DISCUSSION TITLE
A Dropdown multiselect component

### DISCUSSION THREAD

**Discussion by ollejernstrom** [mantinedev/mantine#3592] [Feature requests]
The one component i think mantine is missing is some sort of dropdown component with a multi select feature that is often used for filtering and such. Something 

<img width="270" alt="image" src="https://user-images.githubusercontent.com/113042755/220581334-2462f92c-b430-4126-85a2-8bdd2bec6eb9.png">
Or 
<img width="235" alt="image" src="https://user-images.githubusercontent.com/113042755/220581612-0be31df5-9395-428b-b536-b6e51426f5fb.png">

Is this being built or can it be done using current components?

**Comment by ansonkao** [comment]
You can definitely compose this using existing components in Mantine, and that would probably be preferable over trying to support all possible permutations of use cases in an official component.
